tests/by-util/test_expr.rs

```diff
@@ -274,11 +274,6 @@ fn test_length_mb() {
 
 #[test]
 fn test_regex() {
```

Comment:
why did you remove it ? 
i prefer tests in tests/by-util/test_expr.rs then in syntax_tree.rs
thanks